tests/Fixer/ClassNotation/ModifierKeywordsFixerTest.php

```diff
@@ -981,6 +1054,26 @@ class Foo
                 PHP,
         ];
 
+        yield 'asymmetric visibility with set-visibility before general visibility, omit general public visibility' => [
+            <<<'PHP'
+                <?php class Foo
+                {
+                    public(set) Bar $a;
```

Comment:
As I understood, the idea of the option is to not have superfluous code, shouldn't this one be `public`?

Reply:
I'm not entirely sure whether replacing `public(set)` with `public` should depend solely on the new configuration option.

One could argue that, even with `['omit_public_visibility_set_visibility_present' => false]`, there's superfluous code that could (or should) be removed:

```diff
public private(set) string $foo;
 
- public public(set) string $bar;
+ public string $bar;
```

From a quick glance at PER‑CS 3.0, nothing explicitly addresses this case.

Since this is my very first contribution, I’d welcome any input on the desired outcome and whether it belongs in the scope of this PR or should be handled in a follow‑up. :)